karaf/features/src/main/history/dependencies.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<features xmlns="http://karaf.apache.org/xmlns/features/v1.3.0" name="org.apache.brooklyn-0.9.0-SNAPSHOT">
+    <feature version="0.0.0">
```

Comment:
This is generated by karaf-maven-plugin, and it is generated in src rather than target for the purpose of controlling changes within runtime dependencies. Arguments within pom.xml can get rid of it if we find it is not useful.